1-exercises/I-string-replace/exercise.js

```diff
@@ -13,7 +13,7 @@
 let story =
   "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";
 
-let result = story.replace("", "");
+let result = story.replace("I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.", "I like cats. One night I went to the park and I saw 100000 cats. It was a brilliant night.");
```

Comment:
can you replace only the words that need replacing, instead of the whole sentence?